pkg/processor/runtime/nodejs/test/outputter/outputter.js

```diff
@@ -15,60 +15,60 @@ limitations under the License.
 */
 
 // We use non default file name and handler name to test configuration as well
-exports.testHandler = function(context, event) {
-    if (event.method != 'POST') {
+exports.testHandler = function (context, event) {
+    if (event.method !== 'POST') {
         context.callback(event.method);
-	return;
+        return;
     }
 
-    var body = event.body.toString();
+    const body = event.body.toString();
 
     switch (body) {
-	case 'return_string':
-	    context.callback('a string');
-	    return;
-	case 'return_status_and_string':
-	    context.callback([201, 'a string after status']);
-	    return;
-	case 'return_list':
-	    context.callback([{a: 1}, {b: 2}]);
-	    return;
-	case 'return_status_and_dict':
-	    context.callback([201, {a: 'dict after status', b: 'foo'}]);
-	    return;
-	case 'log':
-	    context.logger.debug('Debug message');
-	    context.logger.info('Info message');
-	    context.logger.warn('Warn message');
-	    context.logger.error('Error message');
-	    context.callback([201, 'returned logs']);
-	    return;
-	case 'log_with':
-	    context.logger.errorWith('Error message', {source: 'rabbit', weight: 7});
-	    context.callback([201, 'returned logs with']);
-	    return;
-	case 'return_response':
-	    headers = event.headers;
-	    headers['h1'] = 'v1';
-	    headers['h2'] = 'v2';
+        case 'return_string':
+            context.callback('a string');
+            return;
+        case 'return_status_and_string':
+            context.callback([201, 'a string after status']);
+            return;
+        case 'return_list':
+            context.callback([{a: 1}, {b: 2}]);
+            return;
+        case 'return_status_and_dict':
+            context.callback([201, {a: 'dict after status', b: 'foo'}]);
+            return;
+        case 'log':
+            context.logger.debug('Debug message');
+            context.logger.info('Info message');
+            context.logger.warn('Warn message');
+            context.logger.error('Error message');
+            context.callback([201, 'returned logs']);
+            return;
+        case 'log_with':
+            context.logger.errorWith('Error message', {source: 'rabbit', weight: 7});
+            context.callback([201, 'returned logs with']);
+            return;
+        case 'return_response':
+            let headers = event.headers;
+            headers['h1'] = 'v1';
+            headers['h2'] = 'v2';
```

Comment:
`h1` and `h2` adhere to the JS identifier constraints.
No need to use brackets notation (`[]`) here to declare new properties.
You can use the simpler dot notation (`.`):
```suggestion
            headers.h1 = 'v1';
            headers.h2 = 'v2';
```